src/views/screens/home/template.njk

```diff
@@ -12,6 +13,27 @@
 
   {{ quickSearch() }}
 
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m">
+        Questionnaire search
+      </h2>
+      <p class="govuk-body">
+        A set of questions to narrow down your search for data and information.<br> 
+        This may be useful if you're not sure what you're looking for.
+      </p>
+    </div> 
+  </div>
+
+  <form action="/guided-search" method="get" role="guided-search">  
```

Comment:
Instead of using the HTTP GET method with `form` to navigate to another URL. We can simply use the `href` attribute of `govukButton` to generate the `<a />`.
Since we do not have any input fields, I feel that using the `<a />` tag instead of the `form` submission is better for navigating to another URL.